From 7a74b2b059f8a6155f41b4b25ec8e162f71492e3 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Fri, 5 Sep 2014 18:56:11 -0400
Subject: [kernel] kexec, purgatory: add clean-up for purgatory directory

Message-id: <1409943373-12669-37-git-send-email-vgoyal@redhat.com>
Patchwork-id: 91542
O-Subject: [RHEL7.1 PATCH 36/38] kexec: purgatory: add clean-up for purgatory directory
Bugzilla: 852066
RH-Acked-by: Dave Anderson <anderson@redhat.com>
RH-Acked-by: Baoquan He <bhe@redhat.com>
RH-Acked-by: Dave Young <dyoung@redhat.com>
RH-Acked-by: Chao WANG <chaowang@redhat.com>

BZ:852066

commit b0108f9e93d0d39050eaa11358852f349bdccb71
Author: Michael Welling <mwelling@ieee.org>
Date: Fri, 29 Aug 2014 15:19:13 -0700

Without this patch the kexec-purgatory.c and purgatory.ro files are not
removed after make mrproper.

Signed-off-by: Michael Welling <mwelling@ieee.org>
Acked-by: Vivek Goyal <vgoyal@redhat.com>
Cc: Ingo Molnar <mingo@elte.hu>
Cc: "H. Peter Anvin" <hpa@zytor.com>
Cc: Thomas Gleixner <tglx@linutronix.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/Makefile b/arch/x86/Makefile
index 5496d66..99892de 100644
--- a/arch/x86/Makefile
+++ b/arch/x86/Makefile
@@ -231,6 +231,7 @@ archclean:
  $(Q)rm -rf $(objtree)/arch/x86_64
  $(Q)$(MAKE) $(clean)=$(boot)
  $(Q)$(MAKE) $(clean)=arch/x86/tools
+ $(Q)$(MAKE) $(clean)=arch/x86/purgatory
 
 define archhelp
   echo  '* bzImage      - Compressed kernel image (arch/x86/boot/bzImage)'
-- 
1.7.1